instrumentation/camel-2.20/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apachecamel/decorators/KafkaSpanDecorator.java

```diff
@@ -66,8 +66,9 @@ public void pre(
     if (partition != null) {
       if (camelDirection == CamelDirection.OUTBOUND) {
         attributes.put(SemanticAttributes.MESSAGING_KAFKA_DESTINATION_PARTITION, partition);
-      } else {
-        attributes.put(SemanticAttributes.MESSAGING_KAFKA_SOURCE_PARTITION, partition);
+        // TODO (trask) does this have a replacement?
+        // } else {
+        // attributes.put(SemanticAttributes.MESSAGING_KAFKA_SOURCE_PARTITION, partition);
       }
```

Comment:
the whole point of removing `source` namespace was to write

```java
attributes.put(SemanticAttributes.MESSAGING_KAFKA_DESTINATION_PARTITION, partition);
```

instead of 

```java
if (camelDirection == CamelDirection.OUTBOUND) {
  attributes.put(SemanticAttributes.MESSAGING_KAFKA_DESTINATION_PARTITION, partition);
} else {
   attributes.put(SemanticAttributes.MESSAGING_KAFKA_SOURCE_PARTITION, partition);
}
```

enjoy ;)